src/Uno.UI/UI/Xaml/Controls/Border/Border.cs

```diff
@@ -247,7 +247,11 @@ public Brush BorderBrush
 
 		private static Brush GetBorderBrushDefaultValue() => SolidColorBrushHelper.Transparent;
 
-		[GeneratedDependencyProperty(ChangedCallback = true, Options = FrameworkPropertyMetadataOptions.ValueInheritsDataContext)]
+		[GeneratedDependencyProperty(ChangedCallback = true, Options = FrameworkPropertyMetadataOptions.ValueInheritsDataContext
+#if __WASM__
+			| FrameworkPropertyMetadataOptions.AffectsArrange
+#endif
+			)]
 		public static DependencyProperty BorderBrushProperty { get; } = CreateBorderBrushProperty();
```

Comment:
TODO:

- Same change for Panel and ContentPresenter?
- I think BorderThickness should also be marked as affects arrange in Border, Panels, and ContentPresenter.